packages/ubuntu_provision/lib/src/locale/locale_page.dart

```diff
@@ -14,6 +14,7 @@ class LocalePage extends ConsumerWidget with ProvisioningPage {
   Future<bool> load(BuildContext context, WidgetRef ref) async {
     final model = ref.read(localeModelProvider);
     await model.init();
+    throw Exception('Test Error');
```

Comment:
For testing purposes this exception is thrown when the language selection page is loaded.